addons/l10n_es_pos/tests/test_frontend.py

```diff
@@ -37,3 +39,43 @@ def get_number_of_regular_invoices():
         num_of_regular_invoices = get_number_of_regular_invoices() - initial_number_of_regular_invoices
         self.assertEqual(num_of_simp_invoices, 3)
         self.assertEqual(num_of_regular_invoices, 1)
+
+    def test_spanish_pos_invoice_no_certificate(self):
+        self.partner_a.vat = "ESA12345674"
+        self.partner_a.country_id = self.env.ref("base.es").id
+        self.partner_a.email = "email@gmail.com"
```

Comment:
a single `write` is more efficient than updating individual fields I believe
```suggestion
        self.partner_a.write({
        'vat': "ESA12345674",
        'country_id': self.env.ref("base.es").id,
        'email': "email@gmail.com"
    })
```